src/Host/Client/Impl/Extensions/SessionExtensions.cs

```diff
@@ -0,0 +1,35 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+using System.Threading.Tasks;
+using Microsoft.Common.Core;
+using Microsoft.R.Host.Client.Session;
+
+namespace Microsoft.R.Host.Client {
+    public static class SessionExtensions {
```

Comment:
We already have class called `RSessionExtensions`
It is in the `Microsoft.R.Host.Client.Session`. We probably should merge them.